build.sbt

```diff
@@ -14,6 +14,8 @@ val unusedOptions = Def.setting(
 
 lazy val scalatraSettings = Seq(
   organization := "org.scalatra",
+  fork in Test := true,
+  baseDirectory in Test := file("."),
```

Comment:
workaround for sbt bug 😢 
- https://github.com/scala/scala-parser-combinators/issues/197
- https://github.com/sbt/sbt/issues/4609